src/components/Form.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function Home() {
+  return (
+    <div className="form">
+      <title>Suggest a Donation Link!</title>
+      <iframe
+        title="Donation Link Google Form"
+        src="https://docs.google.com/forms/d/e/1FAIpQLSdvsS4TEJiDKwphNyaMeoBa4d8XbjsIvQd2hjucI1Ev5p5q5w/viewform?embedded=true"

```

Comment:
Nit: delete extra line. Is the code linted?

Reply:
Linter I'm using actually enforces this styling